.cirrus.yml

```diff
@@ -50,7 +60,7 @@ linux_task_template: &LINUX_TASK_TEMPLATE
     fingerprint_script:
       - wget --quiet https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
       - echo "${CIRRUS_OS} $(sha256sum miniconda.sh)"
-      - echo "$(date +%Y).$(($(date +%U) / ${CACHE_PERIOD})):${CONDA_CACHE_BUILD}"
+      - echo "$(date +%Y).$(($(echo $(date +%U) | sed 's/^0*//') / ${CACHE_PERIOD})):${CONDA_CACHE_BUILD}"
```

Comment:
This is interesting...

The `$(data +%U)` returns `week number of year, with Sunday as first day of week (00..53)`... as it happens, at this point of writing, this is `08`. However this is being interpreted by `bash` as an octal number, which is out of range :scream: ... so this is a fix. On my.

Reply:
Weirdly, as it turned out, it was easy for me to disable all `cirrus-ci` tasks but one, and test the fix without hogging resource unnecessarily.

Serendipity, is a wonderful thing...

Reply:
However `expr` is available... :+1: